incubating/wrappers/java/src/test/java/io/seldon/wrapper/api/TestMessages.java

```diff
@@ -0,0 +1,21 @@
+package io.seldon.wrapper.api;
+
+import static io.seldon.wrapper.util.TestUtils.readFileFromAbsolutePathOrResources;
+
+final public class TestMessages {
+
+  /**
+   * All possible fields based on the SeldonMessage Proto:
+   *   https://docs.seldon.io/projects/seldon-core/en/v1.6.0/reference/apis/prediction.html
+   */
+  public static final String TF_DATA = readFile("requests/defaultData.json");
+  public static final String DEFAULT_DATA = TF_DATA;
+  public static final String JSON_DATA = readFile("requests/jsonData.json");
+  // TODO: add binData
+  // TODO: add strData
+  // TODO: add customData
+
```

Comment:
A collection of all possible payloads for SeldonMessage.
This way all possible requests are in one location and easy to see.
This allows us to easily test all possible assemblies of SeldonMessage.